Project.toml

```diff
@@ -38,6 +38,7 @@ SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
 SimpleDifferentialOperators = "7ccca996-40d6-5c2f-be7f-4eb2934af9c4"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
tests shouldn't depend on DifferentialEquations instead use the components like OrdinaryDiffEq directly.

Reply:
No tests even use OrdinaryDiffEq anymore; I'll remove it now.

Reply:
There should be some integration tests?

Reply:
Whoops yes.  Just removing `DifferentialEquations`, not `OrdinaryDiffEq`.  Meant to say that the test that formerly used `DifferentialEquations` now doesn't use anything of the sort.